FAQ/en.md

```diff
@@ -0,0 +1,61 @@
+# Frequently Asked Questions
+
+## General
+
+### What state is Quaver currently in?
+
+Quaver is currently in the closed Alpha state. In order to use online features of Quaver, you need to sign up for the Alpha.
+
+### Where can I sign up for the Quaver Alpha?
+
+We announce sign-ups before they open, but until then, you will have to wait. Sign-ups are being held during specific time periods, and are not open at all time.
```

Comment:
`Registrations for the closed alpha occur in rounds. Each round of sign-ups are announced before they open.`

Reply:
Applied